Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG PYTHON_VERSION
-FROM python:${PYTHON_VERSION}-slim-buster
+FROM python:${PYTHON_VERSION}-slim-bookworm
```

Comment:
just curious, why do we change it to `bookworm`

Reply:
there's no `3.12-slim-buster` ([only an RC published 6 months ago](https://hub.docker.com/layers/library/python/3.12-rc-slim-buster/images/sha256-bef98ea2e06e1bc3c860b8a3ab089fe6063acb013aada5050a998cc04da368f0?context=explore)). 

Also, debian buster hit no longer getting security, as per https://www.debian.org/releases/buster/. Bookworm is the latest debian version, which the official python images use as [base image](https://hub.docker.com/layers/library/python/slim-bookworm/images/sha256-c127a8c4aca8a5d3ac3a333cbab4c082c7ddbd0891441cc4e30d88dc351f1ce5?context=explore).